src/app/plugins/tictactoe.plugin.ts

```diff
@@ -43,7 +43,11 @@ export class TicTacToe extends Plugin {
   }
 
   private async _createGame(message: IMessage, oppMember: GuildMember) {
-    const game = new TTTGame(message.author, oppMember.user);
+    const game = new TTTGame(
+      message.author,
+      oppMember.user,
+      message.mentions.members?.first()?.roles.cache.some((role) => role.name === 'Chatbot')
```

Comment:
There is a `MemberUtils.hasRole(member:GuildMember, roleName:string)`

Reply:
Probably better to make sure that `message.author.id === this.container.clientService.id`
That way in case there is ever someone else with Chatbot, it won't mess it up

Reply:
This is much easier, thanks!

Reply:
We don't really care about the author, so I swapped to `oppMember.id === this.container.clientService.user?id`